homework_lesson_5/index_lesson_5_mouse.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width,initial-scale=1">
+    <title> base page </title>
+    <meta name="author" content="Anna Danielkiewicz">
+    <meta name="description" content="The base page we will be using for learning javascript">
+    <link rel="stylesheet" href="style_lesson_5_mouse.css" type="text/css">    
+  </head>
+  <body>
+
+<header>  
+
+
+  
+  <div class="strangeButton">
+
+   <button id="strangeButton"> Strange button </button>
+
+    <script type="text/javascript">
+
+    let strangeButtonCmp = document.getElementById("strangeButton");
+    const initialLabel = "Strange button";
+    strangeButtonCmp.classList.add("based-color");
+    
+    function clickHandler(event){
+      console.log('Hey, you just pressed strange button');
+    }
+    
+
+    function mouseOverHandler(event){
+      console.log('Mouse here!'); 
+      strangeButtonCmp.innerText = "What just happened?"
+      strangeButtonCmp.classList.add('inverted-color'); 
```

Comment:
You can unite inverted-color and big-button in one css class, then you will have to add/remove only 1 class at a time